tests/kubernetes/data.py

```diff
@@ -0,0 +1,152 @@
+from tests.kubernetes.utils import get_metrics_from_doc, get_dims_from_doc
+
+# list of monitor definitions
+# the monitor should be a YAML-based dictionary which will be used for the signalfx-agent agent.yaml configuration
+MONITORS_WITHOUT_ENDPOINTS = [
```

Comment:
All of this should go into separate monitor tests in the `tests/monitors` dir.  We might need to start putting subdirs in there for specific monitors if there are multiple files per monitor.